server/apps/embed.js

```diff
@@ -233,10 +256,12 @@ const renderEmbed = ({
         allAtBrowserCombinations,
         title: queryTitle || title || 'Pattern Not Found',
         pattern,
+        priorities,
         phase,
         allBrowsers,
```

Comment:
There's a lot of dead code still in this file, like this variable is not used anywhere for example. Would it be possible to simplify this file a bit? It's currently getting pretty intimidating. Acknowledging of course that I contributed more than anyone to the current situation.

Reply:
So yes the variable appears 7 times in this file, but all of those uses are only related to setting the variable, not using it. Then if you look at the helpers file and main.hbs you'll see that this variable is passed around a bit but never used. That's what I mean when I say it's dead code.

The larger issue, beyond this one variable, is that a lot of the code in this file is passed around and set but never actually utilized, and I think it's definitely worth the effort to overhaul the whole file and remove all the dead code.